package.json

```diff
@@ -183,6 +183,7 @@
     "content-disposition": "0.5.3",
     "core-js": "^3.6.5",
     "custom-event-polyfill": "^0.3.0",
+    "cypress-pipe": "^2.0.0",
```

Comment:
This and the cypress-promise below look like they should be in the devDeps and not in the regular deps section?